src/scripts/main.js

```diff
@@ -1,3 +1,24 @@
 'use strict';
 
-// write your code here
+const table = document.querySelector('table');
+
+const rows = Array.from(table.rows);
+
+rows.forEach(row => {
+  if (row.cells.length > 1) {
+    const secondCell = row.cells[1];
+    if (secondCell) {
+      const clonedCell = secondCell.cloneNode(true);
+      if (row.cells.length > 2) {
+        const lastCell = row.cells[row.cells.length - 1];
+        if (lastCell) {
+          row.insertBefore(clonedCell, lastCell);
+        } else {
+          row.appendChild(clonedCell);
+        }
+      } else {
+        row.appendChild(clonedCell);
+      }
```

Comment:
You are cloning the second cell and then checking if there are more than two cells in the row. This is a bit inefficient because you are cloning the cell even if you don't need to. A more efficient approach would be to first check if there are more than two cells in the row and only then clone the second cell.